subcommands/keys/ca_show.go

```diff
@@ -26,6 +29,7 @@ func init() {
 	}
 	caCmd.AddCommand(cmd)
 	cmd.Flags().BoolVarP(&prettyFormat, "pretty", "", false, "Display human readable output of each certificate")
+	cmd.Flags().BoolVarP(&justRoot, "just-root", "", false, "Only show the Factory root CA certificate")
```

Comment:
Then we need `just-tls` and `just-device-ca`; otherwise, it is unfair.

Reply:
i'll split it out. I started to do this and I remember why i got lazy - mutually exclusive flags aren't released in Cobra yet - https://github.com/spf13/cobra/pull/1654/commits/c437f2ca77a308e612d1ace6295d42c0791c7ac7.

anway - i can do it by hand. just a little inconvenience.